packages/db-migrations/databases/fxa/patches/patch-144-145.sql

```diff
@@ -0,0 +1,213 @@
+-- UPDATE dbMetadata SET value = '143' WHERE name = 'schema-patch-level';
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('144');
+
+-- These columns support a version 2 auth password. Version 2 auth passwords
+-- are created using a different client side stretching approach, where
+-- we used more iterations and a salt that does not require the user email
+-- to make it unique.
+--
+-- These changes result in a different authPW value, and in order to maintain
+-- kB we must also derive a specific wrapKb from the existing kB value. These
+-- additional columns are therefore required.
+--   - clientSalt holds a unique salt per user that's needed in order for users
+--     to do the initial stretch and construct an encrypted password
+--   - verifyHashVersion2 represents the second stretch of this new password provided
+--     to us
+--   - wrapWrapKb represents the second stretch of a wrapKb that results in the
+--     same kB as a wrapWrapKb would create, expect it works with a V2 auth password.
+--
+
+
+ALTER TABLE fxa.accounts ADD COLUMN clientSalt VARCHAR(128);
+ALTER TABLE fxa.accounts ADD COLUMN verifyHashVersion2 BINARY(32);
+ALTER TABLE fxa.accounts ADD COLUMN wrapWrapKbVersion2 BINARY(32);
```

Comment:
A single ALTER will do here.  Also can we put the new columns next to the existing key stretching / password related columns?

nit: we've been using `` `table_name` `` instead of `schema.table_name` in previous migrations.

Reply:
I think it's better to append them, as this is technically non breaking, since it's possible to reference a column by name or position, although, I think we always use name... but still.

I can make this a single alter.

Reply:
> I think it's better to append them ...

True, since we are not on MySQL 8.0.29 yet.